src/OpenTK.Mathematics/Matrix/Matrix4.cs

```diff
@@ -1991,7 +1991,7 @@ private static unsafe void InvertSse3(in Matrix4 mat, out Matrix4 result)
 
             if (MathF.Abs(detM.GetElement(0)) < float.Epsilon)
             {
-                throw new InvalidOperationException("Matrix is singular and cannot be inverted.");
+                MathHelper.ThrowInvalidOperationException("Matrix is singular and cannot be inverted.");
```

Comment:
Previously trying to inline `Matrix4.InvertSse3` would fail with "too many IL bytes" with this change its gets inlined, because it is marked `AggressiveInlining` for some reason. Do you know why? However since this function is never called directly but through `Matrix4.Invert` it doesnt actually get inlined.

Reply:
Ah if `Matrix4.Invert` gets inlined we don't need the `InvertSse3` inline. We don't want to inline the inversion routine itself.

Reply:
Nice. If you want this in pre15 I wont be able to make a PR for it today...

Reply:
Removing the AggressiveInline